precompile/fee_config_manager.go

```diff
@@ -96,8 +101,14 @@ func (c *FeeConfigManagerConfig) Equal(s StatefulPrecompileConfig) bool {
 // Configure configures [state] with the desired admins based on [c].
 func (c *FeeConfigManagerConfig) Configure(chainConfig ChainConfig, state StateDB, blockContext BlockContext) {
 	// Store the initial fee config into the state when the fee config manager activates.
-	if err := StoreFeeConfig(state, chainConfig.GetFeeConfig(), blockContext); err != nil {
-		panic(fmt.Sprintf("fee config should have been verified in genesis: %s", err))
+	if c.InitialFeeConfig != nil {
+		if err := StoreFeeConfig(state, *c.InitialFeeConfig, blockContext); err != nil {
+			panic(fmt.Sprintf("invalid feeConfig provided: %s", err))
```

Comment:
can we add a comment that explains this panic will not occur since verify is passed?

Reply:
done